torch_frame/data/dataset.py

```diff
@@ -149,10 +162,16 @@ def __init__(
             # in-place sorting of col_names for each stype
             self._col_names_dict[stype].sort()
 
-        self.col_to_sep = canonicalize_col_to_sep(
+        self.col_to_sep = canonicalize_col_to_pattern(
             col_to_sep,
             self.col_names_dict.get(torch_frame.multicategorical, []))
 
+        self.col_to_time_format = (None if col_to_time_format is None else
+                                   canonicalize_col_to_pattern(
+                                       col_to_time_format,
+                                       self.col_names_dict.get(
+                                           torch_frame.timestamp, [])))
```

Comment:
In this canonicalization, can we transform `None` to 
`{'col1': None, 'col2': None}` as well?